src/user/getAchievementsEarnedBetween.test.ts

```diff
@@ -76,8 +80,9 @@ describe("Function: getAchievementsEarnedBetween", () => {
         consoleName: "PlayStation Portable",
         cumulScore: 40,
         badgeUrl: "/Badge/193797.png",
-        gameUrl: "/game/3571"
+        gameUrl: "/game/3571",
+        type: "progression"
       }
-    ]);
+    ] as DatedUserAchievement[]);
```

Comment:
These `as T` casts may be misleading.

It seems for this to be impactful, we need to use annotations instead, eg:

```ts
// ASSERT
const expectedResponse: DatedUserAchievement[] = [
  { ... }
];

expect(response).toEqual(expectedResponse);
```

Here's a simple example for why using `as T` assertions may be considered harmful:

```ts
type Fruit = "apple" | "banana";

// Now, we'll assert that `unknownFruit` is of type `Fruit`.
// TypeScript is totally fine with this.
const unknownFruit = "pizza";
const castedFruit = unknownFruit as Fruit;

// Now, even though `castedFruit` has a type of `Fruit`,
// its actual value is "pizza", which is not of the `Fruit` union.
```

Reply:
@wescopeland good point. Updated the PR to be more explicit and removed type casting.